pkg/app/app_test.go

```diff
@@ -225,24 +224,21 @@ func TestInfraConfigRequest(t *testing.T) {
 
 	res, _ := http.Get(ts.URL + "/v1/zones/foo/hosts/bar/infra_config")
 
-	expectedStatus := http.StatusOK
-	if res.StatusCode != expectedStatus {
-		t.Errorf("unexpected status code <<%d>>, want: %d", res.StatusCode, expectedStatus)
+	if diff := cmp.Diff(http.StatusOK, res.StatusCode); diff != "" {
+		t.Errorf("status mismatch (-expected +got):\n%s", diff)
 	}
-
 	rBody, err := io.ReadAll(res.Body)
 	if err != nil {
 		t.Fatal(err)
 	}
-	var actualInfraConfig apiv1.InfraConfig
-	err = json.Unmarshal(rBody, &actualInfraConfig)
+	var got apiv1.InfraConfig
+	err = json.Unmarshal(rBody, &got)
 	if err != nil {
 		t.Fatal(err)
 	}
-
-	expectedInfraConfig := buildInfraCfg([]string{"foo.com:12345"})
-	if !reflect.DeepEqual(expectedInfraConfig, actualInfraConfig) {
-		t.Errorf("Unexpeced infra config. Expected:%#v, Actual:%#v", actualInfraConfig, expectedInfraConfig)
+	expected := buildInfraCfg([]string{"foo.com:12345"})
+	if diff := cmp.Diff(expected, got); diff != "" {
+		t.Errorf("status mismatch (-expected +got):\n%s", diff)
```

Comment:
nit: s/status/config.

Reply:
Umm.. I couldn't understand the reason why you mentioned `s/status/config`. I'll merge this PR but leave this comment open. Could you reply? @ser-io

Reply:
no problem, I incorrectly assumed Github would highlight `status` and it doesn't. The comment was to substitute `status` with `config` in the error message, it would be.

```
t.Errorf("config mismatch (-expected +got):\n%s", diff)
```

Reply:
Ah! Sorry. I made a new pull request for that. https://github.com/google/cloud-android-orchestration/pull/290